packages/fetchai/contracts/erc1155/tests/test_contract.py

```diff
@@ -28,7 +28,18 @@
 
 import pytest
 from aea_ledger_ethereum import EthereumCrypto
+from aea_ledger_ethereum.test_tools.constants import (
+    ETHEREUM_ADDRESS_ONE,
+    ETHEREUM_ADDRESS_TWO,
+    ETHEREUM_PRIVATE_KEY_PATH,
+    ETHEREUM_PRIVATE_KEY_TWO_PATH,
+    ETHEREUM_TESTNET_CONFIG,
+)
+from aea_ledger_ethereum.test_tools.fixture_helpers import (  # noqa  # pylint: disable=unsed-import
+    ganache,
+)
```

Comment:
The reason why defined the fixture in `aea_ledger_ethereum.test_tools.fixture_helpers`, imported here and used `pytest.mark.usefixtures` is because pytest picks up fixtures from the root dir eg `open-aea/test/conftest.py` or `AEA_PROJECT/tests/conftest.py` and that kinda violates the separation of concern since the package tests will depend on that conftest file.